composer.json

```diff
@@ -23,5 +23,8 @@
         "mozilla/wp-sw-manager": "dev-master",
         "marco-c/wp_serve_file": "dev-master"
     },
+    "autoload": {
+      "classmap": ["vendor/"]
```

Comment:
The classes of the dependencies should be defined if they don't support the PSR-4 standard.